src/main/java/mate/academy/lib/DataProcessingException.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.lib;
+
+public class DataProcessingException extends RuntimeException {
+    private final String message;
+    private final Throwable ex;
+
+    public DataProcessingException(String message, Throwable ex) {
+        this.message = message;
+        this.ex = ex;
+    }
```

Comment:
RuntimeException already has these fields. 
```suggestion
    public DataProcessingException(String message, Throwable ex) {
        super(...);
    }
```